src/AutopilotApi/AutopilotClientAwareTrait.php

```diff
@@ -29,6 +30,10 @@ public function getClient(): Client
             return $this->autopilotClient;
         }
 
+        if (getenv('TERMINUS_IS_TESTING_ENV')) {
```

Comment:
Isn't there a better way to do dependency injection for this in tests?

Reply:
Tried to find a better way, this is least impactful

Reply:
Are you testing the `phar`, or is this only for unit tests?